.github/workflows/sanity.yml

```diff
@@ -25,13 +25,13 @@ jobs:
       - name: Install build requirements
         run: sudo apt-get update && sudo apt-get install -y libsasl2-dev libldap2-dev libssl-dev libxmlsec1-dev
 
-      - name: Install python 3.11
+      - name: Install python 3.12
         uses: actions/setup-python@v4
         with:
-          python-version: 3.11
+          python-version: 3.12
 
       - name: Install requirements
-        run: pip3.11 install -r requirements/requirements_all.txt -r requirements/requirements_dev.txt
+        run: pip3.12 install -r requirements/requirements_all.txt -r requirements/requirements_dev.txt
```

Comment:
We hit issues with xmlsec/lxml versions in gateway at runtime. If we see these issues here we might need to apply a similar fix, please ping the gateway team if it happens.

Reply:
And I'm aware of the issue with xmlsec and lxml, which need to be rebuilt from source. 
I was wondering if it would make more sense to build them here in DAB instead of in the component repositories, but I'm not sure that's even possible.